txnkv/transaction/txn.go

```diff
@@ -957,6 +957,11 @@ func (txn *KVTxn) CancelAggressiveLocking(ctx context.Context) {
 	if txn.aggressiveLockingContext == nil {
 		panic("Trying to cancel aggressive locking while it's not started")
 	}
+
+	defer func() {
```

Comment:
We could add some comments about it, maybe we could restore it after the more elegant solution is merged.